Makefile

```diff
@@ -22,54 +22,70 @@ LOGGING_IMAGE_STREAM?=stable
 OPERATOR_NAMESPACE=openshift-operators-redhat
 DEPLOYMENT_NAMESPACE=openshift-logging
 REPLICAS?=0
+OCP_VERSION=4.5
```

Comment:
Where is this from and why is it needed.  @bparees is working to remove any kind of version dependency

Reply:
This is a hindsight that belongs to #285 to copy the CRD files to the manifest folder.